### PR TITLE
Notificatio-ui-issue-fixed

### PR DESCRIPTION
**Pull request by Nikitajoshi39**
| Before | After |
| -------- | -------- |
| before | !notification |

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@Nikitajoshi39 
There should be a slightly space between the search box and the Notification text.

**Comment by Sanjaybhattwebkul** [comment]
@Nikitajoshi39 
This is **working fine**, please find the **screenshot** [Here](https://prnt.sc/tlipF4e09vtc)

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss [hunk 1]
```diff
@@ -935,9 +995,17 @@ body {
...
     .content-container {
         .content {
             .page-header {
+                .control-group {
+                    width: 90%;
+                    display: inline-block;
+                    margin-bottom: 0;
+                    margin-left: 20px;
+                    margin-top:24px;
+                    }
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss:1008]: Remove extra space before this operator

Comment (jitendra-webkul) [reply packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss:1008]: Also add new line after {

File: packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss [hunk 2]
```diff
@@ -949,9 +1017,17 @@ body {
...
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss:1030]: Remove extra space before this operator and add new line too

**Comment by Nikitajoshi39** [comment]
Changes Done.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**